plenum/test/client/test_client.py

```diff
@@ -54,6 +55,7 @@ def testClientShouldNotBeAbleToConnectToNodesNodeStack(pool):
 
 
 # noinspection PyIncorrectDocstring
+@pytest.mark.skip(reason='get rid of registry pool')
 def testSendRequestWithoutSignatureFails(pool):
```

Comment:
I think we need a similar test based on SDK to check that we handle missing signatures properly